Bundle/UKCore-Bundle-MichaelJonesRequest-Example_v3_message.json

```diff
@@ -555,30 +554,39 @@
 				"status": "active",
 				"intent": "order",
 				"category": [
-					{ 
+					{
```

Comment:
Can you remove this completely? The malignant neoplasm code is not a category, and we already have a code for the servicerequest.